aio_pika/tools.py

```diff
@@ -304,6 +304,8 @@ async def __task_inner(self, *args: Any, **kwargs: Any) -> None:
 
             try:
                 await self.callback(*args, **kwargs)
+            except Exception:
+                log.exception("Callback %r error", self)
```

Comment:
you forget reraise it probably

Reply:
If we reraise it, then we are back to square one with asyncio complaining about `Task exception was never retrieved`.

This exception is never caught or logged anywhere except when the GC eventually destroys the `OneShotCallback` object, as explained in the commit message.

Reply:
ah. yep you're right

Reply:
I can confirm that this PR does resolve the "Task exception was never retrieved." error I'm seeing in another context.